Lib/test/test_list.py

```diff
@@ -171,5 +171,30 @@ def test_preallocation(self):
         self.assertEqual(iter_size, sys.getsizeof(list([0] * 10)))
         self.assertEqual(iter_size, sys.getsizeof(list(range(10))))
 
+    def test_count_index_remove_crashes(self):
+        # The count(), index(), and remove() methods were not holding strong
```

Comment:
I would suggest to include a reference to the issue in the comment as this is a regression test.

Reply:
I've added the BPO number to the comment.